teacher.rb

```diff
@@ -0,0 +1,13 @@
+require_relative './person'
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(specialization, *args, **kwargs)
```

Comment:
- [ ] There is no need to process hashes or arrays on this class, you only need to use the same attribute that you are using for the `Person` class plus the `specialization` attribute, so it should be more like this:
```ruby
def initialize(age, specialization, name = "Unknown", parent_permission = true)

Reply:
done :+1: